test/services/permissions.test.js

```diff
@@ -404,257 +559,1036 @@ describe('services > permissions', () => {
+                data: {
+                  collections: {
+                    Users: {
+                      collection: {},
+                      actions: {
+                        save: {
+                          triggerEnabled: [1],
+                        },
+                      },
+                    },
+                  },
+                },
+              });
+
+              const smartActionParameters = {
+                actionName: 'save',
+                userId: 1,
+              };
+
+              const result = await new PermissionsChecker('envSecret', 1).checkPermissions('Users', 'actions', smartActionParameters);
+              expect(result).toBeUndefined();
+            });
+
+            it('should refuse not allowed users', async () => {
+              expect.assertions(1);
+
+              PermissionsChecker.resetExpiration(1);
+              PermissionsChecker.cleanCache();
+              nock.cleanAll();
+              nockObj.get('/liana/v3/permissions?renderingId=1').reply(200, {
+                meta: { rolesACLActivated: true },
+                data: {
+                  collections: {
+                    Users: {
+                      collection: {},
+                      actions: {
+                        save: {
+                          triggerEnabled: [1],
+                        },
+                      },
+                    },
+                  },
+                },
+              });
+
+              const smartActionParameters = {
+                actionName: 'save',
+                userId: 2,
+              };
+
+              await expect(new PermissionsChecker('envSecret', 1).checkPermissions('Users', 'actions', smartActionParameters))
+                .rejects.toThrow("'actions' access forbidden on Users");
+            });
+          });
+
+          describe('handling the user triggering the action', () => {
+            it('should handle the user id as string', async () => {
+              expect.assertions(1);
+
+              PermissionsChecker.resetExpiration(1);
+              PermissionsChecker.cleanCache();
+              nock.cleanAll();
+              nockObj.get('/liana/v3/permissions?renderingId=1').reply(200, {
+                meta: { rolesACLActivated: true },
+                data: {
+                  collections: {
+                    Users: {
+                      collection: {},
+                      actions: {
+                        save: {
+                          triggerEnabled: [1],
+                        },
+                      },
+                    },
+                  },
+                },
+              });
+
+              const smartActionParameters = {
+                actionName: 'save',
+                userId: '1',
+              };
+
+              const result = await new PermissionsChecker('envSecret', 1).checkPermissions('Users', 'actions', smartActionParameters);
+              expect(result).toBeUndefined();
```

Comment:
What is that test 😅 ? Are you testing that it does not throw ?

Reply:
Yes sorry I copied it from the existing tests... I guess it is just checking that the permission is allowed, so that it doesn't throw indeed!
I'll change them all.